src/eventshub/src/component/SearchItem/VendorSearchItem.tsx

```diff
@@ -8,7 +8,7 @@ import { CheckIcon } from '@heroicons/react/16/solid';
 type VendorSearchItemProps = {
   hit: AlgoliaHit<{
     name?: string;
-    stanford_service_provider?: string;
+    stanford_service_provider?: boolean;
```

Comment:
The data type changed on Algolia/Airtable but our check for a value on line 41 doesn't need to change.